album.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
change this file name to index.html as this will be the index of the site.

Reply:
Then your site is not like for like as the ticket mentions
https://www.notion.so/Album-Feature-Layout-6471e0b0452f4393baeaba15921478ee

Reply:
## changed album.heml as index.html
## use 'display:flex;  justify-content: center;' in header, container and footer